Modules/expat/loadlibrary.c

```diff
@@ -88,7 +88,7 @@ HMODULE _Expat_LoadLibrary(LPCTSTR filename)
 
   /* Attempt to find LoadLibraryEx() which is only available on Windows 2000
      and above */
-  pLoadLibraryEx = (LOADLIBRARYEX_FN) GetProcAddress(hKernel32, LOADLIBARYEX);
+  pLoadLibraryEx = (LOADLIBRARYEX_FN) GetProcAddress(hKernel32, LOADLIBRARYEX);
```

Comment:
I verified with recursive grep that these are the only four instances of LOADLIBARYEX in any .c file or any .py file in lib or tools.  Note that the spelling is LOADLIBRARYEX_FN (with 2nd R) on line 51.